administration/package.json

```diff
@@ -57,8 +58,9 @@
     "typescript": "^4.9.5"
   },
   "scripts": {
-    "start": "set -ae; REACT_APP_APPLICATION_COMMIT=$(./application_commit.sh); react-scripts start",
-    "build": "set -ae; REACT_APP_APPLICATION_COMMIT=$(./application_commit.sh); react-scripts build",
+    "start": "set -ae; REACT_APP_APPLICATION_COMMIT=$(./application_commit.sh);react-scripts start",
+    "build": "set -ae; REACT_APP_APPLICATION_COMMIT=$(./application_commit.sh);react-scripts build",
+    "build:staging": "set -ae; REACT_APP_APPLICATION_COMMIT=$(./application_commit.sh);env-cmd -f .env.staging react-scripts build",
```

Comment:
Hmm... I feel like it's a bit overkill to have a new npm command (and a new dev dependency) just for setting the api url.
Can't we infer the API url from the administration's hostname?
(Similar to what you do in https://github.com/digitalfabrik/entitlementcard/pull/921)?

Reply:
Sure we can. But we can just assume that it's `staging. *`